x-pack/plugin/sql/qa/server/build.gradle

```diff
@@ -38,73 +38,68 @@ subprojects {
     apply plugin: 'elasticsearch.java'
   }
 
-
   if (project.name != 'security') {
     // The security project just configures its subprojects
-    apply plugin: 'elasticsearch.legacy-java-rest-test'
-
-    testClusters.matching { it.name == "javaRestTest" }.configureEach {
-      testDistribution = 'DEFAULT'
-      setting 'xpack.ml.enabled', 'false'
-      setting 'xpack.watcher.enabled', 'false'
-    }
-
-
-  dependencies {
-    configurations.javaRestTestRuntimeClasspath {
-      resolutionStrategy.force "org.slf4j:slf4j-api:1.7.25"
-    }
-    configurations.javaRestTestRuntimeOnly {
-      // This is also required to make resolveAllDependencies work
-      resolutionStrategy.force "org.slf4j:slf4j-api:1.7.25"
+    apply plugin: 'elasticsearch.internal-java-rest-test'
+    tasks.named('javaRestTest') {
+      usesDefaultDistribution()
     }
 
-    /* Since we're a standalone rest test we actually get transitive
-     * dependencies but we don't really want them because they cause
-     * all kinds of trouble with the jar hell checks. So we suppress
-     * them explicitly for non-es projects. */
-    javaRestTestImplementation(project(':x-pack:plugin:sql:qa:server')) {
-      transitive = false
+    dependencies {
+      configurations.javaRestTestRuntimeClasspath {
+        resolutionStrategy.force "org.slf4j:slf4j-api:1.7.25"
+      }
+      configurations.javaRestTestRuntimeOnly {
+        // This is also required to make resolveAllDependencies work
+        resolutionStrategy.force "org.slf4j:slf4j-api:1.7.25"
+      }
+
+      /* Since we're a standalone rest test we actually get transitive
+       * dependencies but we don't really want them because they cause
+       * all kinds of trouble with the jar hell checks. So we suppress
+       * them explicitly for non-es projects. */
+      javaRestTestImplementation(project(':x-pack:plugin:sql:qa:server')) {
```

Comment:
Nothing below this line is new - only fixed indentation (was tripping me up, haha).